workers.py

```diff
@@ -140,21 +103,22 @@ def worker() -> None:
             num_trees,
             sample_size,
         )
-        result = isolation_forest.predict(data_frame)
-        host_by_id = {x['id']: x for x in batch_data['results']}
+        results = isolation_forest.predict(
+            data_frame,
+            min_score=env['min_score'],
+        )
 
-        LOGGER.info('Analysis have %s rows in scores', len(result))
+        LOGGER.info('Analysis have %s rows in scores', len(results))
 
         # Build response JSON
         output = {
             'id': batch_id,
             'ai_service': env['ai_service'],
             'data': {
                 'account_number': account_id,
-                'hosts': compile_scores(result, host_by_id),
```

Comment:
Ok, it looks like you took the `hosts` key out

Reply:
Yeah, the forth-coming UI/consumer will need to re-establish with `rad` on the format.